ebuild-writing/text.xml

```diff
@@ -31,4 +31,5 @@ with some general notes and extended examples.
 <include href="misc-files/"/>
 <include href="user-submitted/"/>
 <include href="common-mistakes/"/>
+<include href="bundled-dependencies/"/>
```

Comment:
Can you use a shorter name please, like `bundled-deps`?

(There's of course nothing fundamentally wrong with the current filename, but with the hierarchical structure filenames tend to become very long, and so far we've tried to countermand this by using short path components.)

Reply:
Sorry about the 17 pedantic comments, but you've explicitly asked for a "review of the formatting". :smile:

Reply:
I was in two minds about "dependencies" vs "deps", and I wasn't sure if the abbreviation of "deps" was undesirable somehow. I'll change it.